src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+{# wf_updated_on: 2017-01-27 #}
+{# wf_featured_image: /web/updates/images/2017/01/css-grid/grid.png #}
+{# wf_tags: css,layout,grid #}
+{# wf_featured_snippet: CSS Grid is a new layouting system for the web. #}
+
+# CSS Grid – Table layout is back. Be there and be square. {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+> *TL;DR:* If you are familiar with Flexbox, Grid should feel familiar.
+[Rachel Andrew](https://twitter.com/Rachelandrew) maintains a great
+[website dedicated to CSS Grid](http://gridbyexample.com/) to help you get
+started. Grid is now available in Google Chrome!
+
+## Flexbox? Grid?
+
+Over the past few years, CSS Flexbox has become widely used and
+[browser support](http://caniuse.com/#feat=flexbox) is looking really good
+(unless you are one of the poor souls that have to support IE9 and below).
+Flexbox made a lot of complex layout tasks easier, like equi-distant spacing
+between elements, top-to-bottom layouts or the holy grail of CSS wizardry:
+vertical centering.
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/alignproblem.png" alt="There's no way to align elements across multiple flexbox containers.">
+
+But alas, screens commonly have a second dimension we need to worry about.
+Short of taking care of sizing the elements yourself, sadly, you can't have
+*both* a vertical and horizontal rhythm by just using flexbox alone. This is
+where CSS Grid comes to the rescue.
+
+CSS Grid has been in development, behind a flag in most browsers for over
+*5 years* now and extra time has been spent on interoperability to avoid the
+buggy launch like Flexbox had. So if you use Grid to implement your layout in
+Chrome, chances are you'll get the same result in Firefox and Safari. At the
+time of writing, Microsoft's Edge implementation of Grid is out of date (the
+same as was already present in IE11!) and the update is
+"[under consideration](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/gridupdate/)".
+
+Despite the similarities in concept and syntax, don't think of Flexbox and Grid
+as competing layout techniques. Grid arranges in two dimensions, while Flexbox
+lays out in one and the both have a lot of synergy when used together.
+
+## Defining a grid
+
+To get familiar with the individual properties of Grid I heartily recommend
+\Rachel Andrew’s [Grid By Example](http://gridbyexample.com) or CSS Tricks'
+[Cheat Sheet](https://css-tricks.com/snippets/css/complete-guide-grid/). If you
+are familiar with Flexbox, a lot of the properties and their meaning should be
+familiar.
+
+Let's take a look at a standard 12-column grid layout. The classic 12-column
+layout is very popular as the number 12 is divisible by 2, 3, 4 and 6 and is
+therefore useful for a lot different designs. Let's implement this layout:
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/examplelayout.png" alt="There's no way to align elements across multiple flexbox containers.">
```

Comment:
Don't include widths here, if you want this floated left or right, use `class="attempt-left|right"`